packages/kbn-slo-schema/src/rest_specs/composite_slo.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+import {
+  budgetingMethodSchema,
+  compositeSloIdSchema,
+  dateType,
+  rollingTimeWindowSchema,
+  summarySchema,
+  tagsSchema,
+  targetSchema,
+  weightedAverageCompositeMethodSchema,
+  weightedAverageSourceSchema,
+} from '../schema';
+
+const createCompositeSLOParamsSchema = t.type({
+  body: t.intersection([
```

Comment:
Double checking: If I add extra parameters to the request, should I get this:
```
{"statusCode":400,"error":"Bad Request","message":"Invalid request payload JSON format"}
```

When I used the provided create curl request, I got `Invalid value undefined supplied to`, I am trying to understand when I should expect which io-ts validation error message.